foundation/plugins/org.jboss.tools.foundation.core/src/org/jboss/tools/foundation/core/credentials/internal/CredentialDomain.java

```diff
@@ -60,7 +60,7 @@ public CredentialDomain(Preferences pref) throws BackingStoreException {
 
 		credentials = new HashMap<String, String>();
 		String usersList = pref.get(PROPERTY_USER_LIST, (String)null);
```

Comment:
I now it is too late, but why not this:
``` java
...
		credentials = new HashMap<String, String>();
  		String usersList = pref.get(PROPERTY_USER_LIST, "");
		if( !usersList.isEmpty()) {
  			String[] users = usersList.split("\n");
  			for( int i = 0; i < users.length; i++ ) {
  				credentials.put(users[i], NOT_LOADED_PASSWORD);
...
```
no null checks, no extra ternary operator.